dotnet/src/SemanticKernel.UnitTests/Functions/KernelFunctionFromPromptTests.cs

```diff
@@ -118,6 +118,7 @@ public async Task ItFailsIfInvalidServiceIdIsProvidedAsync()
         var exception = await Assert.ThrowsAsync<KernelException>(() => kernel.InvokeAsync(func));
 
         // Assert
-        Assert.Equal("Service of type Microsoft.SemanticKernel.AI.TextGeneration.ITextGenerationService and names service3 not registered.", exception.Message);
+
+        Assert.Equal("Service of type Microsoft.SemanticKernel.AI.TextGeneration.ITextGenerationService with one of the following serviceIds: service3 or modelIds:  not registered.", exception.Message);
```

Comment:
Maybe it would be better if exception won't contain `or modelIds: ` part if `modelIds` is empty.

Reply:
Yep, I'll fix that later